e2e/test/iothub/SasCredentialAuthenticationTests.cs

```diff
@@ -96,13 +94,21 @@ public async Task JobClient_Http_SasCredentialAuth_Success()
             try
             {
                 // act
-                JobResponse createJobResponse = await jobClient
+                ScheduledTwinUpdate twinUpdate = new ScheduledTwinUpdate
+                {
+                    QueryCondition = query,
+                    Twin = twin,
+                    StartTimeUtc = DateTime.UtcNow
+                };
+                ScheduledJobsOptions scheduledTwinUpdateOptions = new ScheduledJobsOptions
+                {
+                    JobId = jobId,
+                    MaxExecutionTimeInSeconds = (long)TimeSpan.FromMinutes(2).TotalSeconds
```

Comment:
Can we change this property to be a TimeSpan instead of long with a specified unit of seconds?